includes/wikia/nirvana/WikiaParserTagController.class.php

```diff
@@ -69,7 +69,7 @@ protected function registerResourceLoaderModules( Parser $parser ) {
 		return $markerId;
 	}
 
-	public final function onParserAfterTidy( Parser &$parser, &$text ) {
+	public final function onParserAfterTidy( Parser $parser, &$text ) {
```

Comment:
Replaced the few final occurrences of those. I have also executed complete SUS Selenium suite + X-Wing Selenium test suite against the changeset - time to merge.